docs/ancestry.md

```diff
@@ -1264,13 +1265,87 @@ print(tree.tmrca(0,1))
 
 ### Selective sweeps
 
-```{eval-rst}
-.. todo:: Document the selective sweep models.
+Msprime provides the option to perform coalescent approximations
+to selective sweeps, in which a beneficial mutation moves through
+the population. This is done through the use of a structured 
+coalescent model in the spirit of 
+[Braverman et al. (1995)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206652/).
+
+Looking backwards in time the population enters a sweep phase where
+the population is split between two classes of selective backgrounds,
+those linked to the beneficial allele, call it {math}`B`, and those not,
+{math}`b`. During this portion of the simulation we track competing
+rates of coalescence and recombination on the {math}`B` and {math}`b`
+backgrounds.
+
+This implementation is reasonable general, althougth there are 
+current limitations (e.g., no change of population size during
+a sweep). The user supplies a final allele frequency and a starting
+allele frequency, between which msprime will simulate a stochastic 
+sweep trajectory according to a conditional diffusion model [(Coop
+and Griffiths, 2004](https://pubmed.ncbi.nlm.nih.gov/15465123/);
+[Kim and Stephan 2002](https://pubmed.ncbi.nlm.nih.gov/11861577/)).
+
+Beyond the start and end frequencies of the sweep trajectory, the user
+must specify the strength of selection as measured by 
+{math}`\alpha = 2N_es` where s is the selective
+advantage of the {math}`B` homozygote over the {math}`b` homozygote
+and {math}`h=0.5`. The position represents the location along
+the chromosome where the beneficial allele occurs.
+ All other parameters can be set as usual. 
```

Comment:
perhaps add something like "and where `N_e` is the effective population size during the transit of the sweep (which affects both coalescent rate, as usual, as well as stochasticity of the allele frequency trajectory)"?